dts/bindings/led_strip/led-strip.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2019, Linaro Limited
```

Comment:
In the commit message, maybe you mean "color ordering property" instead of "ordering property" ?

Reply:
Updated